libexec/config-lxc

```diff
@@ -10,4 +10,10 @@ if [ -z "$LXC_BRIDGE" ]; then
     LXC_BRIDGE=br0
 fi
 
-sed "s;ROOTFS;$wd/target-$LXC_SUITE-$LXC_ARCH;;s;ARCH;$LXC_ARCH;g;;s;GUESTIP;$LXC_GUEST_IP;g;s;GUESTLINK;$LXC_BRIDGE;g" < etc/lxc.config.in > var/lxc.config
+if [[ $(lxc --version) = 3.* ]]; then
```

Comment:
IMO this should be the other way around. if the version is 2 or earlier, it should use the old config. The new config should be the default as that is what will be used going forwards.

Reply:
Instead of using `lxc --version`, I think you should use `lxc-start --version`.

Reply:
Fixed